src/Compilers/VisualBasic/Portable/VBResources.resx

```diff
@@ -5081,7 +5081,8 @@
                                   referenced metadata files. 
                                   import_list:namespace,...
 /langversion:&lt;number&gt;             Specify language version: 
-                                  9|9.0|10|10.0|11|11.0|12|12.0|14|14.0|15|15.0|default
+                                  9|9.0|10|10.0|11|11.0|12|12.0|14|14.0|15|
+                                  15.0|default|latest
```

Comment:
>15.0|default|latest [](start = 34, length = 19)

Crazy nit pick but in the C# help we use Default and Latest, here default and latest (casing difference).